src/screens/Bridge/Exchange-CrossChain/Exchange.tsx

```diff
@@ -289,10 +297,17 @@ function Exchange({ fetchExchangeTitle }: Props) {
 
     const bestOffer = sortedOffers?.find((offer) => !!offer.feeInfo && offer.provider !== EXCHANGE_PROVIDER.LIFI);
 
+    if (hideAllOffers && selectedProvider) {
+      const selectedOffer = sortedOffers?.find((offer) => !!offer.feeInfo && offer.provider === selectedProvider);
```

Comment:
What's the type of sortedOffers? Do we have a defined type or its any?

Reply:
ExchangeOffer[]. it's defined now